subsys/bluetooth/host/conn_internal.h

```diff
@@ -138,6 +141,14 @@ struct bt_conn {
 		struct bt_conn_sco	sco;
 #endif
 	};
+#if defined(CONFIG_BT_REMOTE_VERSION)
+	struct bt_conn_rv {
+	u8_t  version;
```

Comment:
Wrong indentation here for the struct members (should be one more tab)

Reply:
Like this?
```
#if defined(CONFIG_BT_REMOTE_VERSION)
	struct bt_conn_rv {
		u8_t	version;
		u16_t	manufacturer;
		u16_t	subversion;
	struct k_sem rv_sem;
	} rv;
#endif
```

Reply:
No. Everything between `{ ... }` should have the same indentation. I.e. it should look something like this:
```
struct bt_conn_rv {
	u8_t  version;
	u16_t manufacturer;
	u16_t subversion;
	struct k_sem sem;
} rv;
```

Note that there's no reason to have a `rv_` prefix for the semaphore since this is already inside a struct called `rv`